core/src/com/ericc/the/game/entities/Attack.java

```diff
@@ -7,10 +7,11 @@
 import com.ericc.the.game.components.PositionComponent;
 import com.ericc.the.game.components.RenderableComponent;
 import com.ericc.the.game.map.Map;
+import com.ericc.the.game.utils.GridPoint;
 
 public class Attack extends Entity {
-    public Attack(int x, int y, Map map, Direction direction) {
-        add(new PositionComponent(x, y, map));
+    public Attack(GridPoint xy, Map map, Direction direction) {
```

Comment:
Totalnie nie podoba mi sie to nazewnictwo wszędzie _xy_, juz _pos_ byłoby lepsze

Reply:
Ale komponent position się nazywa pos, wtedy odwołanie się do gridPointa wyglądałoby tak: pos.pos :o